run-tests-with-docker.sh

```diff
@@ -10,7 +10,7 @@ sleep $((2+NODES))
 docker-compose run \
                --name verify-tests \
                -e TEST_ENV="${TEST_ENV:-"staging"}" \
-               verify-tests -n $NODES -o "--strict -f pretty -f junit -o testreport/ --tags 'not @ignore' $*"
+               verify-tests -n $NODES -o "--strict -f pretty -f junit -o testreport/ --tags 'not @Eidas' $*"
```

Comment:
As in `pre-commit.sh` should we keep the ability to ignore tests marked `@ignore` too?